services/worker-manager/src/providers/aws.js

```diff
@@ -359,6 +360,10 @@ class AwsProvider extends Provider {
           await this.removeWorker({ worker, reason: 'terminateAfter time exceeded' });
         }
       }
+      const { isZombie, reason } = Provider.isZombie({ worker });
```

Comment:
naming is hard .. but  I think `zombie` is a good analogy, same as zombie-processes

Reply:
I think it is ok too, but we should probably document somewhere what zombie means.